### PR TITLE
partner repacks

### PR DESCRIPTION
**Pull request by escapewindow**

### CODE REVIEW

File: how-tos/partner-repacks.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
+Partner Repacks
+===============
+
+Partner Repacks are where we take a built and signed Firefox desktop build and modify things (branding, plugins, prefs), potentially also do so for l10n repacks, then re-sign the new files. The main point of contact here will be Mike Kaply.
+
+EME Free is a repacked Firefox without Encrypted Media Extensions, which allows you to run a Firefox without DRM (at the cost of broken video streaming).
```
Comment (gbrownmozilla) [inline how-tos/partner-repacks.rst:6]: "For example, ..." might make this less awkward.

File: how-tos/partner-repacks.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+For staging releases, we use the ``moz-releng-automation-stage`` bot's repositories: `staging release partners <https://github.com/moz-releng-automation-stage/repack-manifests>`__, `staging esr partners <https://github.com/moz-releng-automation-stage/esr-repack-manifests>`__, and `staging EME Free <https://github.com/moz-releng-automation-stage/mozilla-EME-free-manifest>`__.
+
+The scripts are in `repack-scripts <https://github.com/mozilla-partners/repack-scripts>`__.
+
+How do we add/remove partners?
+------------------------------
+
+This is via manifest manipulation. If we're adding a new partner, we'll need to create a partner repo first, before pointing to it in the manifest.
+
+This has largely been out of Releng's hands, and we've only seen a reduction of the number of partners since the 2020 layoffs. These PRs have come from Kaply.
```
Comment (gbrownmozilla) [inline how-tos/partner-repacks.rst:24]: Still, referencing an existing bug or PR here might be helpful.

File: how-tos/partner-repacks.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+How do I see partner tasks?
+---------------------------
+
+There are a ton of ``release-partner-repack-*`` and ``release-eme-free-*`` tasks in a given Firefox desktop RC or dot-release promote graph. Also `Firefox desktop betas >= b5 <https://github.com/mozilla-releng/shipit/blob/79bbd4c5b30234c54b238d064074399ea45a8803/api/src/shipit_api/admin/release.py#L99-L106>`__.
+
+Many of these tasks might not visible in Treeherder. We intentionally hid them, then decided to show them, but haven't finished that process. We probably want to make them visible for easier sheriffing of failed release tasks.
```
Comment (gbrownmozilla) [inline how-tos/partner-repacks.rst:31]: s/not visible/not be visible/

**Review by gbrownmozilla — APPROVED** [review]
Thanks!

File: how-tos/partner-repacks.rst (hunk 1 above)
Comment (escapewindow) [reply how-tos/partner-repacks.rst:6]: Maybe? EME free is lumped together with partner repacks, but isn't strictly a partner repack.

File: how-tos/partner-repacks.rst (hunk 3 above)
Comment (jcristau) [reply how-tos/partner-repacks.rst]: Might want to reference https://bugzilla.mozilla.org/show_bug.cgi?id=1568367 here?

File: how-tos/partner-repacks.rst (hunk 1 above)
Comment (escapewindow) [reply how-tos/partner-repacks.rst]: I adjusted the title to make more sense.

**Pull request merged**